backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/RunVmCommandBase.java

```diff
@@ -146,6 +150,7 @@ protected void cleanupPassthroughVnics(Guid hostId) {
     @Override
     public void rerun() {
         decreasePendingVm();
+        getVdsManager().unpinVmCpus(getVmId());
```

Comment:
for migrate-vm it should be getDestinationVdsManager, right?

Reply:
yes, thanks